RULES.md

```diff
@@ -150,6 +151,12 @@ Value of field `route_id` should exist in GTFS data.
 
 Value of field `agency_id` should exist in GTFS data.
 
+<a name="E036"/>
+
+### E036 - `service_id` not found
+
+Value of field `service_id` should exist in GTFS data.
```

Comment:
```suggestion
Value of field `service_id` should exist in GTFS `calendar.txt` or `calendar_dates.txt` data.
```

Reply:
Will report this precision for all the relevant rule descriptions

Reply:
Sounds good - you could probably get rid of the last word `data` too, re-reading it now.